ansible/roles/docker/templates/ubuntu1804_sharedlibs.Dockerfile.j2

```diff
@@ -24,7 +24,7 @@ RUN apt-get update && apt-get install apt-utils -y && \
       git \
       pkg-config \
       curl \
-      python-pip \
+      python 3.8 \
```

Comment:
This doesn't look correct. Either `python3.8` or removed if the default Python 3 on Ubuntu 18.04 is new enough.